### DISCUSSION TITLE
Different translations for element creation and enumeration

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#8762] [Feature Requests]
In element creation procedure in russian language it have to be different word ending:
i.e must be "Создание работЫ" (_Creation of article_):
![image](https://user-images.githubusercontent.com/6277090/137130767-d0291f7d-9913-4d94-83ee-642cf3e7ae4b.png)

In this context (collection display) word ending is fine:
![image](https://user-images.githubusercontent.com/6277090/137130901-7c6cb8d4-5f42-462b-8062-a3a7e28ad6f9.png)

Also I'm appologise, but we have different word endings for 1, 2-4, 5 - 20 number of things.

In example:

1 Одна работА _"One article"_
2 Две работЫ _"Two articles"_
3 Три работЫ  
4 Четыре работЫ 
5 Пять работ _<-- zero ending_
...
20 Двадцать работ
21 Двадцать **одна** работА _"Twenty one articles"_ ("one" is main here)

Directus version **9.0.0.rc-96**

**Comment by rijkvanzanten** [comment]
Directus currently doesn't have a way to set singular vs plural names for collections (see https://github.com/directus/directus/discussions/3069). In the English translations, we're relying on `Creating Item in {collection}` to get around that. Would a similar approach work in Russian?

**Reply by IlyaPetrovM** [reply]
Well. `{Collection} - creating` will be better imho, because "Creating Item in {collection}" puts the collection title in subordinate relationship (this changes word ending in russian).

**Reply by rijkvanzanten** [reply]
Yeah! That's totally fine 🙂

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
